### DISCUSSION TITLE
Support for Rounded Corners Background Color Style

### DISCUSSION THREAD

**Discussion by aibobrov** [gonzalezreal/swift-markdown-ui#336] [Ideas]
Currently inline code fragment (``) is rendered using build-in SwiftUI attribute `backgroundColor`. 
To be more accurate with GitHub theme inline code fragment should be rendered with rounded-corner-background.

It can be done with SwiftUI's new [TextRenderer](https://developer.apple.com/documentation/swiftui/textrenderer) api ([useful article](https://alexanderweiss.dev/blog/2024-06-24-using-textrenderer-to-create-highlighted-text))

**Comment by antonjazz** [comment]
Yes - and it would look much nicer to have a small amount of padding around the text in  the shaded `code` shape.
